code/__DEFINES/misc.dm

```diff
@@ -141,7 +141,7 @@
 //2 = reduced hud (just hands and intent switcher)
 //3 = no hud (for screenshots)
 
-#define MINERAL_MATERIAL_AMOUNT 2000
+#define MINERAL_MATERIAL_AMOUNT 1000
```

Comment:
It was the guy who added RnD in the first place. He decided that one material bar is 2000 cm3 of material. Then the metal and glass, which were 3750 cm3 before, were changed to be 2000 cm3 too. And it caused shit to break, so never ever do that again. Just make a note that one sheet is 2000 material and use that in your "simplified material equations".